node-net-manager/env/ContainerNetDeployment.go

```diff
@@ -12,7 +12,7 @@ import (
 )
 
 type ContainerDeyplomentHandler struct {
-	env *Environment
+	env **Environment
```

Comment:
Why do we need it to be a double-pointer?

Reply:
It was necessary because the Environment was allocated only after the registration happened. 

But I changed it in commit [7c8aa50](https://github.com/oakestra/oakestra-net/pull/31/commits/7c8aa50c4e8bc903d8cce0959bb7c2c1ce6df016), to instead dereference the Environment after it has been initialized. Now only a single indirection is required.